docs/security/host-certs.md

```diff
@@ -1,15 +1,21 @@
 How to Get Host and Service Certificates
 =============================================
 
-This document is for system administrators. After reading this document you should be able to apply for and install a grid certificate on a grid resource. This document does not explain how to apply for a grid user certificate. To apply for a grid user certificate click [here](user-certs.md) instead!
+This document is for system administrators. After reading this document you
+should be able to apply for and install a grid certificate on a grid resource.
+This document does not explain how to apply for a grid user certificate. To
+learn how to apply for a grid user certificate click [here](user-certs.md)
+instead!
 
 Requirements
-=================
+--------------
 
-Before requesting a new host or service certificate, you should check if a certificate is not installed already using **openssl**. In this case you may safely skip this document:
+Before requesting a new host or service certificate, you should check if a
+certificate is not installed already using **openssl**. In this case you may
```

Comment:
Holy ambiguity batman!
I think this means "you should check, using `openssl`, if a certificate is already installed"?

I interpreted this as "you should verify that there isn't already a certificate on the system that was installed by OpenSSL"

Reply:
Also, there's a FAQ that basically repeats this info.  Should I remove that?

Reply:
Yea, I vote removing it from the FAQ